cypress/e2e/admin/administrateAuthentication.feature

```diff
@@ -7,15 +7,15 @@ Feature: Admin can edit system configuration
   Scenario: As an admin i want to see my systems and edit ldap systems of provider: general
     Given I am logged in as a 'admin1_nbc' at 'nbc'
     When I click on administration in menu
-    When I click on sub menu school
+    When I click on sub menu school and see new admin page
```

Comment:
Should be 2 steps:
When I click on submenu
Then I see new admin page

Reply:
`When I navigate to new school admin page via sub menu`